lib/msal-core/src/ScopeSet.ts

```diff
@@ -51,4 +51,19 @@ export class ScopeSet {
   static removeElement(scopes: Array<string>, scope: string): Array<string> {
     return scopes.filter(value => value !== scope);
   }
+
+  /**
+   * Parse the scopes into a formatted scopeList
+   * @param scopes
+   */
+  static parseScope(scopes: Array<string>): string {
+    let scopeList: string = "";
```

Comment:
return scopes.reduce((memo, val, index) => (
    ${memo}${index !== scopes.length - 1 ? `${val} `: val }`
), "")

Reply:
Yeah was going comment same thing, but this is just copy/pasta, so I'm cool with not touching this as part of this PR.

Reply:
+1, this will come in the scopes refactor.